contracts/guilds/lib/math_utils.cairo

```diff
@@ -0,0 +1,81 @@
+%lang starknet
```

Comment:
Can you please write tests for these functions?

There should also be checks for overflows and `uint256_check` in various places in this lib.